core/Policy/CompliancePolicy.php

```diff
@@ -42,8 +42,17 @@ abstract class CompliancePolicy implements SystemSettingInterface, MeasurableSet
     use ConfigGetterTrait;
 
     abstract public static function getName(): string;
-    abstract public static function getDescription(): string;
     abstract public static function getTitle(): string;
+    abstract protected static function generateDescription(): string;
+
+    public static function getDescription(): string
+    {
+        $description = static::generateDescription();
+
+        Piwik::postEvent('CompliancePolicy.updatePolicyDescription', [static::class, &$description]);
```

Comment:
Technically it doesn't matter, but the established usage pattern across the codebase is to pass the variable that can be updated by the event as the first parameter and any other data as subsequent parameters.

```suggestion
        Piwik::postEvent('CompliancePolicy.updatePolicyDescription', [&$description, static::class]);
```